frontend/index.d.ts

```diff
@@ -73,7 +73,23 @@ interface CAPIType {
  * this data could eventually be defined in dotcom-rendering
  */
 interface ConfigType {
-    ajaxUrl: string
+    ajaxUrl: string;
+}
+
+interface GADataType {
```

Comment:
What are your thoughts on defining types here vs in actual packages/co-located with relevant code? There's a go aphorism I quite like which is: 'interfaces should be defined at the point of use.'

Reply:
Yeah, it's not ideal. I following the pattern of other "global" objects extracted from the request body sent from frontend. There's a [card to break down `parse-capi`](https://trello.com/c/AfMx5JSx/179-refactor-parse-capi), perhaps we can sort out the types in this file then too?